web/themes/new_weather_theme/templates/partials/daily-forecast-list-item.html.twig

```diff
@@ -0,0 +1,50 @@
+{#
+This partial expects an array of forecast period
+assoc arrays (as formatted by the WeatherDataService),
+usually of size 2 corresponding to both daytime and
+nighttime periods.
+
+In some cases -- like the case of the current day -- there might
+only be a nighttime period.
+#}
+<li class="display-block margin-bottom-5">
+  <div class="daily-date-and-temp display-flex flex-row flex-justify flex-align-center margin-bottom-2">
+    <time class="display-block flex-1 text-bold font-heading-md" datetime="{{ periods[0].startTime }}">
+      {% if dayLabel %}
+        <span>{{ dayLabel }},</span>
+      {% else %}
+        <span>{{ periods[0].dayName }}</span>
+      {% endif %}
+      <span class="display-block">{{ periods[0].monthAndDay }}</span>
+    </time>
+
+    {% for period in periods %}
+      {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'period': period } %}
+    {% endfor %}
+  </div>
+  {% for period in periods %}
+    {% set label =  period.isDaytime ? "Day" | t : "Night" | t %}
+    <div class="daily-forecast-summary-area margin-bottom-2">
+      <span class="display-block text-bold font-body-xs">
+        {{label}}
+      </span>
+      <div class="display-flex flex-row flex-align-start">
```

Comment:
the alignemnt looks a little off with short lines, could we change this one to `flex-align-center`